src/rust/ensogl/README.md

```diff
@@ -0,0 +1,285 @@
+EnsoGL is a blazing fast vector rendering engine that comes batteries included:
+ - Rich set of primitive shapes, including circle, rectangle, rectangle with rounded corners,
```

Comment:
These docs will be improving when I'll be writing final report for one of our grants this week.